runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -71,8 +71,13 @@ default Color getRangePrayerColor()
 		description = "XP drop color when a mage prayer is active",
 		position = 3
 	)
-	default Color getMagePrayerColor()
-	{
-		return new Color(0x15, 0x80, 0xAD);
-	}
+	default Color getMagePrayerColor() { return new Color(0x15, 0x80, 0xAD); }
+
+	@ConfigItem(
+			keyName = "default Color",
+			name = "Default Color",
+			description = "XP drop color when combat prayer is inactive",
+			position = 4
+	)
+	default Color getDefaultColor() { return null; }
```

Comment:
Check chatColors config I believe we just dont add method body for no value colors